test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/ValidateCongestionPricingAfterReconnect.java

```diff
@@ -0,0 +1,215 @@
+								.providingFeeTo(normalPrice::set).logging(),
+						getAccountBalance(GENESIS)
+								.setNode(reconnectingNode)
+								.unavailableNode().logging()
+				).when(
+						/* update the multiplier to 10x with a 1% congestion for tmpMinCongestionPeriodInSecs */
+						fileUpdate(APP_PROPERTIES)
+								.fee(ONE_HUNDRED_HBARS)
+								.payingWith(EXCHANGE_RATE_CONTROL)
+								.overridingProps(Map.of(
+										"fees.percentCongestionMultipliers", "1,10x",
+										"fees.minCongestionPeriod", tmpMinCongestionPeriodInSecs
+								)).logging(),
+						fileUpdate(THROTTLE_DEFS)
+								.fee(ONE_HUNDRED_HBARS)
+								.payingWith(EXCHANGE_RATE_CONTROL)
+								.contents(artificialLimits.toByteArray()).logging(),
+						blockingOrder(
+								IntStream.range(0, 20).mapToObj(i ->
+										contractCall(oneContract)
+												.payingWith(GENESIS)
+												.fee(ONE_HUNDRED_HBARS)
+												.sending(ONE_HBAR)
+												.logging())
+										.toArray(HapiSpecOperation[]::new)
+						)
+				).then(
+						withLiveNode(reconnectingNode)
+								.within(5 * 60, TimeUnit.SECONDS)
+								.loggingAvailabilityEvery(30)
+								.sleepingBetweenRetriesFor(10),
+						blockingOrder(
+								IntStream.range(0, 10).mapToObj(i ->
+										contractCall(oneContract)
+												.payingWith(GENESIS)
+												.fee(ONE_HUNDRED_HBARS)
+												.sending(ONE_HBAR)
+												.setNode(reconnectingNode)
+												.logging())
+										.toArray(HapiSpecOperation[]::new)
+						),
+						contractCall(oneContract)
+								.payingWith(civilianAccount)
+								.fee(ONE_HUNDRED_HBARS)
+								.sending(ONE_HBAR)
+								.via("pricyCallAfterReconnect")
+								.setNode(reconnectingNode)
+								.logging(),
+
+						getTxnRecord("pricyCallAfterReconnect")
+								.payingWith(GENESIS)
+								.providingFeeTo(tenXPrice::set)
+								.setNode(reconnectingNode)
+								.logging(),
+
+						/* check if the multiplier took effect in the contract call operation */
+						withOpContext((spec, opLog) -> {
+							Assert.assertEquals(
+									"~10x multiplier should be in affect!",
+									10.0,
+									(1.0 * tenXPrice.get()) / normalPrice.get(),
+									0.1);
+						}),
+
+						/* revert the multiplier before test ends */
+						fileUpdate(THROTTLE_DEFS)
+								.fee(ONE_HUNDRED_HBARS)
+								.payingWith(EXCHANGE_RATE_CONTROL)
+								.contents(defaultThrottles.toByteArray())
+								.setNode(reconnectingNode)
+								.logging(),
+						fileUpdate(APP_PROPERTIES)
+								.fee(ONE_HUNDRED_HBARS)
+								.payingWith(EXCHANGE_RATE_CONTROL)
+								.overridingProps(Map.of(
+										"fees.percentCongestionMultipliers", defaultCongestionMultipliers,
+										"fees.minCongestionPeriod", defaultMinCongestionPeriod
+								))
+								.logging(),
+
+						cryptoTransfer(HapiCryptoTransfer.tinyBarsFromTo(GENESIS, FUNDING, 1))
+								.payingWith(GENESIS).logging()
+				);
```

Comment:
Just curious, all these transactions from beginning you have the `.logging()` enabled, is this required for validator or optional?

Reply:
@ljianghedera  As the test is very small, I have enabled logging to get an idea easily by seeing the logs. It is optional , I will remove the logging

Reply:
If you haven't, please keep them, since there won't be a large amount of log messages.